packages/ui-position/src/mirrorPlacement.ts

```diff
@@ -32,13 +42,17 @@ const mirror = {
   offscreen: 'offscreen'
 }
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'placement' implicitly has an 'any' type... Remove this comment to see the full error message
-function mirrorPlacement(placement, delimiter) {
+function mirrorPlacement<D extends string | undefined = undefined>(
```

Comment:
Is this typed correctly?
When I try the examples in the bottom of this file they give me TS errors:
```
mirrorPlacement("start end") // error
mirrorPlacement('top start', ' ') // error
mirrorPlacement(["start", "end"]) // OK
```

Reply:
True, I accidentally deleted something during a refactor. I even made tests for the 2 mirror utils, they didn't have one before.